gradio/components/annotated_image.py

```diff
@@ -64,7 +63,9 @@ def __init__(
         visible: bool = True,
         elem_id: str | None = None,
         elem_classes: list[str] | str | None = None,
-        **kwargs,
+        render: bool = True,
+        root_url: str | None = None,
```

Comment:
So someone can update the `root_url` from an event? 😅

Reply:
I guess this can happen already?

Reply:
Haha yea I can't think of any reason anyone would want to but yes this is now possible. We need to include this in the constructor since we no longer accept arbitrary *kwarg params, and this is set in `Blocks.from_config`. If we want, we could rename this to be an internal variable `_root_url` (or even better `_remote_url`) but that can be a separate PR I think

Reply:
I think treating it like _skip_init_processing makes sense (also related to spaces)

Reply:
Sounds good -- I've added as a todo here: https://github.com/gradio-app/gradio/issues/5830